beluga_amcl/src/amcl_node.cpp

```diff
@@ -120,6 +120,12 @@ AmclNode::AmclNode(const rclcpp::NodeOptions& options) : BaseAMCLNode{"amcl", ""
     declare_parameter("z_rand", rclcpp::ParameterValue(0.5), descriptor);
   }
 
+  {
+    auto descriptor = rcl_interfaces::msg::ParameterDescriptor();
+    descriptor.description = "If false, Likelihood Field sensor model won't model unknown space.";
```

Comment:
@DPR00 consider having matching descriptions in ROS 1 and ROS 2. I like the ROS 1 one better.